packages/python/accern_xyme/accern_xyme.py

```diff
@@ -1895,8 +1896,12 @@ def read(
             self,
             key: str,
             chunk: Optional[int],
-            force_refresh: bool = False) -> Optional[ByteResponse]:
-        return self.read_blob(key, chunk, force_refresh).get_content()
+            force_refresh: bool = False,
+            filter_id: bool = True) -> Optional[ByteResponse]:
+        content = self.read_blob(key, chunk, force_refresh).get_content()
+        if filter_id and isinstance(content, pd.DataFrame):
+            content = content[~content["row_id"] < 0]
```

Comment:
also add set_index -- like in the backend.
also why ~ and < not >=?

Reply:
set index with which column?